tests/Tokenizer/TokensTest.php

```diff
@@ -1174,6 +1174,21 @@ public function provideRemoveTrailingWhitespaceCases()
         return $cases;
     }
 
+    /**
+     * No action that begins with the word "remove" should add anything to the collection.
+     */
+    public function testRemovingLeadingWhitespaceWillNotIncreaseTokensCount()
+    {
+        $tokens = Tokens::fromCode('<?php
+                                    // Foo
+                                    $bar;');
+        $originalCount = $tokens->count();
+
+        $tokens->removeLeadingWhitespace(4);
+
+        $this->assertLessThanOrEqual($originalCount, $tokens->count());
```

Comment:
`<=` ? doesn't sound strict. what exactly we are asserting here ?

Reply:
That we won't increase the size of token's collection